### ISSUE TITLE
Help context menu item causes error insude of custom blocks

### ISSUE THREAD

**Issue by thomaswp** [jmoenig/Snap--Build-Your-Own-Blocks#1124]
Inside the custom block editor, if a user right-clicks on a block and selects the "Help" item, nothing will happen. The browser throws an exception saying the IDE is null.

![untitled](https://cloud.githubusercontent.com/assets/1750176/12728062/043166ca-c8ee-11e5-8e2c-0a7ead28bce9.png)

Here's the stack trace:
```
Uncaught TypeError: Cannot read property 'resourceURL' of null
BlockMorph.showHelp @ blocks.js:2640
TriggerMorph.trigger @ morphic.js:8350
MenuItemMorph.mouseClickLeft @ morphic.js:8583
HandMorph.processMouseUp	 @ morphic.js:9753
(anonymous function) @ morphic.js:10365

```

**Comment by jmoenig** [comment]
good catch, thanks!

**Comment by cycomachead** [comment]
Fixed with df0269f5ccfce06e3c577aadb5b3559263edb9b2
